src/rnp/rnpcfg.cpp

```diff
@@ -373,6 +373,19 @@ rnp_cfg_getlist(rnp_cfg_t *cfg, const char *key)
     return NULL;
 }
 
+std::string rnp_cfg_getlist_string(const rnp_cfg_t *cfg, const std::string &key, size_t index) {
+    list *lval = rnp_cfg_getlist(const_cast<rnp_cfg_t*>(cfg), key.c_str());
+    if (lval) {
+        if (index < list_length(*lval)) {
+            rnp_cfg_val_t *val = (rnp_cfg_val_t *)list_at(*lval, index);
+            return rnp_cfg_val_getstr(val);
```

Comment:
`rnp_cfg_val_getstr()` may return NULL, which will raise an exception in std::string() constructor. Please check for NULL and return empty string in that case.